buildAllExamples.sh fails on os x

This is a fresh download of OF.

of_v0.9.7_osx_release
OS X 10.11.6 (15G31)
XCode Version 7.3.1 (7D1014)

```
/Users/james/src/of_v0.9.7_osx_release/examples/shader/08_displacementMap/src/ofApp.cpp:19:39: warning: 'getTextureReference' is deprecated [-Wdeprecated-declarations]
    plane.mapTexCoordsFromTexture(img.getTextureReference());
                                      ^
In file included from /Users/james/src/of_v0.9.7_osx_release/examples/shader/08_displacementMap/src/ofApp.cpp:1:
In file included from src/ofApp.h:3:
In file included from ../../../libs/openFrameworks/ofMain.h:64:
../../../libs/openFrameworks/graphics/ofImage.h:317:52: note: 'getTextureReference' has been explicitly marked deprecated here
    OF_DEPRECATED_MSG("Use getTexture",ofTexture & getTextureReference());
                                                   ^
/Users/james/src/of_v0.9.7_osx_release/examples/shader/08_displacementMap/src/ofApp.cpp:27:21: error: no viable conversion from 'ofPixels_<unsigned char>' to 'unsigned char *'
    unsigned char * pixels = img.getPixels();
                    ^        ~~~~~~~~~~~~~~~
/Users/james/src/of_v0.9.7_osx_release/examples/shader/08_displacementMap/src/ofApp.cpp:45:9: warning: 'getTextureReference' is deprecated [-Wdeprecated-declarations]
    img.getTextureReference().bind();
        ^
In file included from /Users/james/src/of_v0.9.7_osx_release/examples/shader/08_displacementMap/src/ofApp.cpp:1:
In file included from src/ofApp.h:3:
In file included from ../../../libs/openFrameworks/ofMain.h:64:
../../../libs/openFrameworks/graphics/ofImage.h:317:52: note: 'getTextureReference' has been explicitly marked deprecated here
    OF_DEPRECATED_MSG("Use getTexture",ofTexture & getTextureReference());
                                                   ^
2 warnings and 1 error generated.

** BUILD FAILED **


The following build commands failed:
	CompileC build/08_displacementMap.build/Debug/08_displacementMap.build/Objects-normal/x86_64/ofApp.o src/ofApp.cpp normal x86_64 c++ com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
failed building ../../examples/shader/08_displacementMap Debug
```

--- a/examples/shader/08_displacementMap/src/ofApp.cpp
+++ b/examples/shader/08_displacementMap/src/ofApp.cpp
@@ -16,7 +16,7 @@ void ofApp::setup(){
     img.allocate(80, 60, OF_IMAGE_GRAYSCALE);
 
     plane.set(800, 600, 80, 60);
-    plane.mapTexCoordsFromTexture(img.getTextureReference());
+    plane.mapTexCoordsFromTexture(img.getTexture());
 }
 
 //--------------------------------------------------------------
@@ -24,7 +24,7 @@ void ofApp::update(){
     float noiseScale = ofMap(mouseX, 0, ofGetWidth(), 0, 0.1);
     float noiseVel = ofGetElapsedTimef();
 
-    unsigned char * pixels = img.getPixels();
+    ofPixels & pixels = img.getPixels();
     int w = img.getWidth();
     int h = img.getHeight();
     for(int y=0; y<h; y++) {
@@ -42,7 +42,7 @@ void ofApp::draw(){
 
     // bind our texture. in our shader this will now be tex0 by default
     // so we can just go ahead and access it there.
-    img.getTextureReference().bind();
+    img.getTexture().bind();
 
     shader.begin();
 

